java/compaction-job-execution/src/test/java/sleeper/compaction/jobexecution/CompactSortedFilesReportingTest.java

```diff
@@ -60,15 +60,16 @@ public void shouldReportJobStatusUpdatesWhenCompacting() throws Exception {
         CompactionJob compactionJob = compactionFactory().createCompactionJob(
                 dataHelper.allFileInfos(), dataHelper.singlePartition().getId());
         dataHelper.addFilesToStateStoreForJob(compactionJob);
+        String taskId = "task-id";
```

Comment:
Can this be passed into createCompactSortedFiles to ensure the correct task ID gets used?